vaccine/tests/test_real411.py

```diff
@@ -18,36 +20,79 @@ async def real411_mock(sanic_client):
     app = Sanic("real411_mock")
     app.requests = []
 
-    @app.route("/form-data", methods=["GET"])
-    def check(request):
-        app.requests.append(request)
-        return response.file_stream(
-            "vaccine/tests/real411.json", mime_type="application/json"
+    @app.route("/complaint-type", methods=["GET"])
+    def complaint_type(request):
+        with open("vaccine/tests/real411.json") as f:
+            data = json.load(f)
+        complaints = [
+            c for c in data["ComplaintType"] if c["code"] == request.args.get("code")
+        ]
+        return response.json(
+            {
+                "success": True,
+                "message": "",
+                "data": {"rows": complaints, "total": len(complaints)},
+                "errors": None,
+            }
+        )
+
+    @app.route("/language", methods=["GET"])
+    def language(request):
+        with open("vaccine/tests/real411.json") as f:
+            data = json.load(f)
+        languages = [
+            c for c in data["Language"] if c["code"] == request.args.get("code")
+        ]
+        return response.json(
+            {
+                "success": True,
+                "message": "",
+                "data": {"rows": languages, "total": len(languages)},
+                "errors": None,
+            }
+        )
+
+    @app.route("/source", methods=["GET"])
+    def source(request):
+        with open("vaccine/tests/real411.json") as f:
+            data = json.load(f)
+        sources = [c for c in data["Source"] if c["code"] == request.args.get("code")]
+        return response.json(
+            {
+                "success": True,
+                "message": "",
+                "data": {"rows": sources, "total": len(sources)},
+                "errors": None,
+            }
         )
 
     @app.route("/file_upload", methods=["PUT"])
     def file_upload(request):
         app.requests.append(request)
         return response.text("")
 
-    @app.route("/submit/v2", methods=["POST"])
+    @app.route("/complaint", methods=["POST"])
     def submit(request):
         app.requests.append(request)
         file_count = len(request.json["file_names"])
         return response.json(
             {
-                "complaint_ref": 1,
+                "success": True,
+                "complaint_ref": "WDM88J4P",
```

Comment:
References were switched to strings, just want to make sure that the tests assert that, in case we assume integer anywhere